### PR TITLE
Updating App Emulation php docs

### PR DESCRIPTION
**Pull request by edward-simpson**
### Description (*)
Correcting PHPDocs for `Emulation::startEnvironmentEmulation()`
Function returns void, does not return initial store environment information as docs suggest

### Manual testing scenarios (*)
None required

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [N/A] All new or changed code is covered with unit/integration tests (if applicable)
 - [N/A] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]
Nice! Well spotted and updated.

**Comment by BarnyShergold** [comment]
@magento run all tests

**Review by Den4ik — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @Den4ik, thank you for the review.
[ENGCOM-9209](https://jira.corp.magento.com/browse/ENGCOM-9209) has been created to process this Pull Request

File: app/code/Magento/Store/Model/App/Emulation.php [hunk 1]
```diff
@@ -111,7 +111,7 @@ public function __construct(
     /**
      * Start environment emulation of the specified store
      *
-     * Function returns information about initial store environment and emulates environment of another store
+     * Function emulates environment of another store
```
Comment (bgorski) [inline app/code/Magento/Store/Model/App/Emulation.php]: Thanks for spotting this!
Can you merge this line with the one above ("Start environment emulation of the specified store") to create a simpler description? Right now we'd be left with two lines that say a very similar thing.

**Review by bgorski — CHANGES_REQUESTED** [review]

File: app/code/Magento/Store/Model/App/Emulation.php (hunk 1 above)
Comment (edward-simpson) [reply app/code/Magento/Store/Model/App/Emulation.php]: @bgorski good shout, that should be better

**Comment by BarnyShergold** [comment]
@magento run all tests

**Review by bgorski — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @bgorski, thank you for the review.
[ENGCOM-9209](https://jira.corp.magento.com/browse/ENGCOM-9209) has been created to process this Pull Request

**Review by bgorski — CHANGES_REQUESTED** [review]
One more thing - sorry for not noticing it before. In order to merge Pull Requests, the code should pass all tests. While functional tests failed because of code not related with the file you touched so you don't have to fix them, static tests report one error that's related to Emulation.php. While not directly related to your docblock improvement, it would still have to be fixed. Can you do that?

**Comment by edward-simpson** [comment]
@bgorski absolutely!  That's now done

**Comment by BarnyShergold** [comment]
@magento run all tests

**Review by bgorski — APPROVED** [review]
@edward-simpson thanks for your contribution! 

Functional tests are failing but that's not caused by this PR as it doesn't introduce any functional changes in the logic of the modified file, so I'm approving the pull request.

**Comment by magento-engcom-team** [comment]
Hi @bgorski, thank you for the review.
[ENGCOM-9209](https://jira.corp.magento.com/browse/ENGCOM-9209) has been created to process this Pull Request

**Comment by BarnyShergold** [comment]
@magento run Functional Tests B2B,Functional Tests CE,Functional Tests EE

**Comment by BarnyShergold** [comment]
@magento run Functional Tests B2B,Functional Tests CE,Functional Tests EE

**Comment by BarnyShergold** [comment]
@magento run Functional Tests B2B,Functional Tests CE,Functional Tests EE

**Comment by BarnyShergold** [comment]
@magento run Functional Tests CE

**Comment by BarnyShergold** [comment]
@magento run Functional Tests CE

**Comment by andrewbess** [comment]
This PR has already been merged

**Pull request merged**